code/game/objects/structures/window.dm

```diff
@@ -599,3 +600,88 @@
 
 /obj/structure/window/reinforced/clockwork/fulltile/unanchored
 	anchored = FALSE
+
+/obj/structure/window/paperframe
+	name = "paper frame"
+	desc = "A fragile separator made of thin wood and paper."
+	icon = 'icons/obj/smooth_structures/paperframes.dmi'
+	icon_state = "frame"
+	dir = FULLTILE_WINDOW_DIR
+	opacity = 1
+	max_integrity = 15
+	obj_integrity = 15
+	fulltile = 1
+	flags = PREVENT_CLICK_UNDER
+	smooth = SMOOTH_TRUE
+	canSmoothWith = list(/obj/structure/window/paperframe, /obj/structure/mineral_door/paperframe)
+	glass_amount = 2
+	glass_type = /obj/item/stack/sheet/paperframes
+	heat_resistance = 233
+	decon_speed = 10
+	CanAtmosPass = ATMOS_PASS_YES
+	resistance_flags = FLAMMABLE
+	armor = list(melee = 0, bullet = 0, laser = 0, energy = 0, bomb = 0, bio = 0, rad = 0, fire = 0, acid = 0)
```

Comment:
Create this list in Initialize before calling ...()

if(!armor)
    armor = blah blah blah
. = ..()